fixtures/vuejs3-typescript/App.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div id="app">
+    <img src="./assets/logo.png">
+    <hello></hello>
+  </div>
+</template>
+
+<script lang="ts">
+import Hello from './components/Hello.vue'
+
+class TestClassSyntax {
+
+}
+
+export default {
+  name: 'app',
+  components: {
+    Hello
+  }
+}
+</script>
+
+<style>
+#app {
+  font-family: 'Avenir', Helvetica, Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  text-align: center;
+  color: #2c3e50;
+  margin-top: 60px;
+}
+</style>
+
+<style lang="scss">
+#app {
+  display: flex;
+  color: #2c3e90;
+}
+</style>
+
+<style lang="less">
+#app {
+  margin-top: 40px;
+}
+</style>
```

Comment:
Typescript in vue2 vs vue3 is different enough that I have 2 separate directories.